style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  display: flex;
+  background: url(background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.wrapper {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  align-content: space-around;
+  margin-top: 2%;
+}
+
+.mylogo {
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 18px;
+  font-weight: bold;
+  line-height: 20px;
+  flex: 1;
+  padding-left: 2%;
+}
```

Comment:
### OPTIONAL

- [ ] Original design has not underline on logo. You can fix it for stick design.

![image](https://user-images.githubusercontent.com/38635158/129232839-f4bede74-d778-4e7b-9adc-53ee8ec1dd00.png)